README.md

```diff
@@ -115,37 +116,36 @@ Install build dependencies appropriate for your OS:
 
 ``` shell
 # Fedora
-sudo dnf install gcc-c++ make automake alsa-lib-devel libpng-devel SDL2-devel \
+sudo dnf install gcc-c++ meson alsa-lib-devel libpng-devel SDL2-devel \
                  SDL2_net-devel opusfile-devel fluidsynth-devel
 ```
 
 ``` shell
 # Debian, Ubuntu
-sudo apt install build-essential automake libasound2-dev libpng-dev \
+sudo apt install build-essential meson libasound2-dev libpng-dev \
                  libsdl2-dev libsdl2-net-dev libopusfile-dev libfluidsynth-dev
 ```
 
 ``` shell
 # Arch, Manjaro
-sudo pacman -S gcc automake alsa-lib libpng sdl2 sdl2_net opusfile fluidsynth
+sudo pacman -S gcc meson alsa-lib libpng sdl2 sdl2_net opusfile fluidsynth
 ```
 
 ``` shell
 # macOS
 xcode-select --install
-brew install autogen automake libpng sdl2 sdl2_net opusfile fluid-synth
+brew install meson libpng sdl2 sdl2_net opusfile fluid-synth
 ```
 
-Compilation flags suggested for local optimised builds:
+Instructions for creating a release build:
 
 ``` shell
 git clone https://github.com/dosbox-staging/dosbox-staging.git
 cd dosbox-staging
-./autogen.sh
-./configure CPPFLAGS="-DNDEBUG" \
-            CFLAGS="-O3 -march=native" \
-            CXXFLAGS="-O3 -march=native"
-make -j$(nproc)
+meson setup -Dbuildtype=release build
+cd build
+ninja
```

Comment:
Question - should we prefer `meson compile` instead of `ninja`, simply because because it's agnostic of the backend (or more specfically, that it will use the OS-specific builder?).

Reply:
`meson compile` is very new sub-command. On Ubuntu 20.04 it's not available - that's why I rewrote README.md to say `ninja`.

From a pragmatic point of view, I prefer `meson compile -C build` because it does not require me to change the directory, but overall I don't think it matters :) I think meson documentation will be shifting to `meson compile` going forward.

Reply:
Ahh; I didn't realize how new it was!
Sounds like we should just mirror Meson's documentation, and right now they're still mentioning `ninja`. 
Works for me :-)